sympy/crypto/crypto.py

```diff
@@ -1266,44 +1266,57 @@ def rsa_private_key(p, q, e):
 
 def encipher_rsa(i, key):
     """
-    Return encryption of ``i`` by computing `i^e` (mod `n`),
-    where ``key`` is the public key `(n, e)`.
+    Return encryption of "i" by computing 'i^e' (mod 'n'),
+    where "key" is the key '(n, e)'. "key" may be a public
+    key or a private key.
 
     Examples
     ========
-
+    *Public Key Encryption*
+    
```

Comment:
There should be no hidden space here, to pass the tests.
I'd recommend you to configure your editor , to trim spaces automatically.